tests/components/resource_request/test_views.py

```diff
@@ -21,6 +21,31 @@
 
 
 class TestResourceRequestViews:
+    async def test_list_resource_requests_filter_200(
```

Comment:
- Please rewrite this with `@pytest.mark.parametrize('parameter', ['username', 'email'])` to have isolated testing of each parameter. Check `test_list_projects_returns_project_filtered_by_parameter_full_match` test case as an example.
```suggestion
    async def test_list_resource_requests_returns_resource_request_filtered_by_parameter_full_match(
```
- Please also move this test case to the very bottom.

Reply:
Done

Reply:
Could you please consider the test case name I've suggested? You can update it in a way you want.
The point is to have better understanding on what is directly tested in the test. Check that `resource_requests_filter_200` is a small part of what the test actually tests.

Reply:
I've made this change. I'm not a fan of the super long names, I thought mine gave sufficient info